api-inference-community/docker_images/adapter_transformers/requirements.txt

```diff
@@ -0,0 +1,3 @@
+starlette==0.14.2
+api-inference-community==0.0.19
+git+https://github.com/Adapter-Hub/adapter-transformers#egg=adapter-transformers
```

Comment:
This could alternatively be `adapter-transformers` no? Or is there any particular reason to install directly from head? (it's usually better to have `adapter-transformers==version` so we can reliably test/debug things)

Reply:
I implemented this using the current dev version of adapter-transformers, including some syntax that isn't released in PyPI yet. This should happen in the coming days, will change to `adapter-transformers==2.2.0` then.

Reply:
SG, thanks!